protocol/0014-ORDT-order_types.md

```diff
@@ -90,6 +90,76 @@ Network orders are used during [position resolution](./0012-POSR-position_resolu
+
+### Iceberg Order Submission
+
+1. A persistent (GTC, GTT, GFA, GFN) iceberg order that is not crossed with the order book is included in the order book with order book volume == initial peak size. No trades are generated.
+2. An iceberg order with either an ordinary or pegged limit price can be submitted.
+3. An iceberg post only order can be submitted.
+4. An iceberg reduce only order is rejected.
+5. Margin requirement for iceberg order - is it based on initial peak size i.e. displayed quantity OR actual quantity ? Assume its displayed quantity - needs confirmation.
+6. For a persistent (GTC, GTT, GFA, GFN) iceberg order that's submitted margin is calculated and deducted correctly both during submission as well as during subsequent refreshes.
+7. For an iceberg order, the orders are refreshed immediately after producing a trade. Every time volume is taken from the displayed quantity , ensure the order is refreshed if if display quantity < minimum peak size.
+   If the order is successfully refreshed , then the order loses its time priority and is pushed to the back of the queue.
+8. For an iceberg order that's submitted when the market is in auction, only the displayed quantity is filled when coming out of auction. What other attributes need to be checked here ?
+
+### Iceberg Order Batch Submission
+
+1. For multiple iceberg orders submitted as a batch of orders with a mix of ordinary limit orders and market orders, the iceberg orders are processed atomically and the order book volume and price, margin calculations , order status are all correct - What else needs checking ?
+2. What other scenarios need testing ?
+
+### Iceberg Order Submission - Negative tests
+
+1. An iceberg order with a non persistent TIF (IOC, FOK) is rejected with a valid error message.
+2. An iceberg market order with any TIF is rejected with a valid error message.
+3. A reduce-only iceberg order with any TIF is rejected with a valid error message.
+4. An iceberg order with initial peak size less than the minimum order size rejected with a valid error message.
+5. An iceberg order with initial peak size greater than the total order size is rejected with a valid error message.
+6. An iceberg order with minimum peak size less than 0 is rejected with a valid error message.
+7. An iceberg order with minimum peak size greater than initial peak size is rejected with a valid error message.
+
+### Iceberg Order Amendment
+
+1. Amending an iceberg order to increase size will increase the total and remaining quantities of the order and time priority of the order is not lost.
+2. Amending an iceberg order to decrease size will decrease the total and remaining quantities and time priority of the order is not lost.
+3. Amend an iceberg order to decrease size so that the displayed quantity is decreased. Total and remaining quantity is decreased, margin is recalculated and released and time priority is not lost.
+4. What other scenarios do we need to consider here ?
```

Comment:
Amending total quantity to less than displayed quantity should also reduce displayed amount